content/docs/app-developer-guide/build-an-app.md

```diff
@@ -22,9 +26,10 @@ app.
 
 When using `pack`, you can run `pack builder suggest` for a list of suggested builders.
 
-```bash
+```
```

Comment:
For my personal learnings, is there a reason why this was removed? Is it incompatible, a preference, or should we be using `shell`?

Reply:
katacoda previously didn't support this. I tested it again and it seems to be working now. Will revert these changes.

Reply:
Will fix this in a subsequent PR